main/build.gradle

```
@@ -58,6 +59,18 @@ repositories {
     mavenCentral()
 }
 
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            groupId = 'org.mobilitydata'
+            artifactId = 'application'
```

I believe this should be:

```suggestion
            artifactId = rootProject.name
```

...to match the manifest in the `jar {}` section. The end result should be `gtfs-validator`.